wdl2cwl/main.py

```diff
@@ -633,7 +655,7 @@ def get_expr_apply(self, wdl_apply_expr: WDL.Expr.Apply) -> str:
             return self.get_expr(only_arg)
         elif function_name == "select_first":
             array_obj = arguments[0]
-            array_items = [self.get_expr(item) for item in array_obj.items]  # type: ignore
+            array_items = [str(self.get_expr(item)) for item in array_obj.items]  # type: ignore
```

Comment:
as the elements of an array might not be strings this converts every item to a string